NAS-102098 doc new column in Tasks->Replication Tasks (#1311)
freenas/freenas-docs: userguide/intro.rst

* The :guilabel:`Name` field has been added to
  :menuselection:`Tasks --> Replication Tasks --> Add`.

> * The *Last Snapshot* column has been added to

Reviewer: ```suggestion
* A :guilabel:`Last Snapshot` column has been added to
```
